documents/customers.md

```diff
@@ -155,6 +155,251 @@ Customer customer = instance.customers.fetch(customerId);
 
 -------------------------------------------------------------------------------------------------------
 
+
+### Add Bank Account of Customer
+
+```java
+String customerId = "cust_N5mywh91sXB69O"
+
+JSONObject customerRequest = new JSONObject();
+customerRequest.put("ifsc_code","UTIB0000194");
+customerRequest.put("account_number","916010082985661");
+customerRequest.put("beneficiary_name","Pratheek");
+customerRequest.put("beneficiary_address1","address 1");
+customerRequest.put("beneficiary_address2","address 2");
+customerRequest.put("beneficiary_address3","address 3");
+customerRequest.put("beneficiary_address4","address 4");
+customerRequest.put("beneficiary_email","random@email.com");
+customerRequest.put("beneficiary_mobile","8762489310");
+customerRequest.put("beneficiary_city","Bangalore");
+customerRequest.put("beneficiary_state","KA");
+customerRequest.put("beneficiary_country","IN");
+
+BankAccount bankaccount = instance.customers.addBankAccount(customerId, customerRequest)
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| customerId*  | string      | Unique identifier of the customer.  |
+| account_number | string      | Customer's bank account number.  |
+| beneficiary_name | string  | The name of the beneficiary associated with the bank account.  |
+| beneficiary_address1 | string      | The virtual payment address.  |
+| beneficiary_email  | string      | Email address of the beneficiary.  |
+| beneficiary_mobile | integer      | Mobile number of the beneficiary.  |
+| beneficiary_city  | string      | The name of the city of the beneficiary.  |
+| beneficiary_state | string      | The state of the beneficiary.  |
+| beneficiary_pin   | interger    | The pin code of the beneficiary's address.  |
+| ifsc_code          | string      | The IFSC code of the bank branch associated with the account.  |
```

Comment:
I see the rows are not aligned properly, can you please check it once?

Reply:
@rohitcbr It might be difficult to align the property because of long description. But in preview, it maintains the alignment

Reply:
got it, +1